locales/cy/translations.json

```diff
@@ -848,5 +848,20 @@
       "yemeni" : "WELSH - Yemeni",
       "zambian" : "WELSH - Zambian",
       "zimbabwean" : "WELSH - Zimbabwean"
+    },
+
+    "month": {
```

Comment:
I'd use the plural here for the key, like we've done for the other two lists - so "months".